doc/vxlan/EVPN/EVPN_VXLAN_HLD.md

```diff
@@ -1147,7 +1173,14 @@ Linux kernel version 4.9.x used in SONiC requires backport of a few patches to s
    +---------+--------------+-------+
    Total count : 1
 
+  7. show vxlan counters
+   +--------+------------+----------+------------+----------+
```

Comment:
We need to add more details on how counters are designed for P2P (in the future ?) and P2MP case.

Reply:
Done